kanbananza-backend/src/main/controllers/user.js

```diff
@@ -24,13 +24,6 @@ const create = async (req, res, next) => {
         })
       );
     }
-
-    if (e instanceof mongoose.Error.ValidationError) {
-      return next(
-        HttpError.fromMongooseValidationError(e, "Invalid user information")
-      );
-    }
-
```

Comment:
@RemiCarriere why was this removed? I think saving the user doocument in the service can still throw mongse validation errors that are not checked so far